drivers/bridge/setup_ipv6.go

```diff
@@ -133,17 +132,12 @@ func setupNDPProxying(config *networkConfiguration, i *bridgeInterface) error {
 			logrus.Warnf("Unable to enable NDP default proxying: %v", err)
 		}
 	}
-
-	// Get current NDP all proxying setup
-	ndpProxyDataAll, err := ioutil.ReadFile(ndpProxyConfAll)
```

Comment:
Do you mean to change https://github.com/mostolog/libnetwork/blob/907282e5195e7bfaf507da0375bcaaf95a618603/drivers/bridge/setup_ipv6.go#L139 ?:

```
    if ndpProxyConfPerm[0] != '1' {
        if err := ioutil.WriteFile(ndpProxyConfPath, []byte{'1', '\n'}, ndpProxyConfPerm); err != nil {
            logrus.Warnf("Unable to enable NDP proxying for interface %s: %v", config.NDPProxyInterface,err)
        }
    }
```